src/tribol/interface/mfem_tribol.cpp

```diff
@@ -82,7 +84,19 @@ void setMfemLORFactor( IndexT cs_id, int lor_factor )
   SLIC_ERROR_ROOT_IF( !coupling_scheme->hasMfemData(),
                       "Coupling scheme does not contain MFEM data. "
                       "Create the coupling scheme using registerMfemCouplingScheme() to set the LOR factor." );
+
+  // reset binning proximity based on original LOR factor
```

Comment:
Explain the rationale here for clarity. It gets confusing when considering HO to LO and scalings with LOR factor. Is the idea that the original binning proximity scale is some percent of the HO element thickness? Then, by dividing it by LOR factor it approximately becomes some percentage of the low order element thickness? Is that what we want? A user may want 2*HO_element_thickness to allow for one HO face to interpenetrate twice the thickness of the opposing HO element, for instance. By dividing by the lor_factor, they won't see contact with that amount of interpen? Since it is a scaling one approach isn't necessarily wrong, but I'm trying to think about what the user may expect when setting this. I think it is best if they don't have to think too much about the LOR mesh.

Reply:
I think moving LOR factor to a Tribol parameter makes sense, just so we can clean up this code.